extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -137,6 +138,10 @@ export class SchemaCompareDialog {
 			};
 		}
 
+		Telemetry.sendTelemetryEvent('SchemaCompareStart', {
+			'sourceIsDacpac': this.sourceIsDacpac.toString(),
+			'targetIsDacpac': this.targetIsDacpac.toString()
+		});
 		let schemaCompareResult = new SchemaCompareResult(sourceName, targetName, sourceEndpointInfo, targetEndpointInfo);
 		schemaCompareResult.start();
 	}
```

Comment:
Do you want to track anything about when this ends?

Reply:
added end times for comparison, script generation, and apply